light/client.go

```diff
@@ -821,12 +850,35 @@ func (c *Client) Witnesses() []provider.Provider {
 	return c.witnesses
 }
 
+// BlacklistedWitnessIDS returns the blacklisted witness IDs.
+//
+// NOTE: providers may be not safe for concurrent access.
+func (c *Client) BlacklistedWitnessIDs() []string {
+	c.providerMutex.Lock()
+	defer c.providerMutex.Unlock()
+
+	witnessIds := []string{}
```

Comment:
```suggestion
	witnessIds := make([]string, 0, len(c.blacklist))
```
small allocation efficiency nit :)   Some linters would prefer this to be `witnessIDs` (I defer to preference on that)